.github/workflows/lint.yml

```diff
@@ -62,3 +64,13 @@ jobs:
         run: |
           source .venv/bin/activate
           pre-commit run --all-files
+      - name: Run graphql-codegen
+        id: graphql-codegen
+        run: |
+          source .venv/bin/activate
+          pre-commit run --all-files
+          cd mlflow/server/js
+          yarn install
+          yarn graphql-codegen
```

Comment:
when do we need to run this?

Reply:
Every time there's any schema change or javascript change, we should make sure this is up-to-date

Reply:
should we run this in the js workflow?

Reply:
Yeah I thought about this but it seems like the js workflow is not blocking? So is that ideal?

Reply:
Actually, I'm going to revert this change in this PR for now, and we can introduce it later, as right now there's not really any graphql query checked-in to the frontend